content/mentoring-sessions.md

```diff
@@ -0,0 +1,33 @@
+# Mentoring Sessions
+
+In Mentoring sessions you get the opportunity to meet and learn from experienced tech industry mentors. These experts are super helpful, and they'll guide you through even the most challenging concepts. Whenever you feel stuck, they'll be right there to assist you, ensuring you level up your skills in no time.
+
+What's even better is that these sessions are conducted with real everyday developers who know what they are doing and are trained to be able to teach you in the clearest way possible. So, any burning questions you have, they've got the answers! Plus, you have the freedom to choose a time that perfectly fits your schedule. 🤓
+
+No matter which modality you go for, you have the chance to clarify any doubts you may have with a real pro in the field! We care deeply about your learning journey, and we believe that when you can get all your questions answered, it'll help you build rock-solid foundations.
+
+That's why we've made sure you can have these mentoring sessions whenever it suits you best. We want your learning experience to be as convenient as possible, so you can focus on growing and killing it out there!
+
+### 📆 Steps to Schedule a Mentoring Session
+
+1. Once you're in your program, look to the right-hand side. There, you'll find the **cohort** section. Right below it, you'll see a block titled **Schedule a 1-1 mentoring session**. That's where you can set up a personal mentoring session.
+
+![Mentoring Sessions](../images/mentoring.sessions.png)
+
+2. When you click on the blue button that says **Schedule mentoring session**, you'll see two options on the screen. Just click on the one that says **GeekPal 1-1 Coding Mentorship**.
```

Comment:
There could be more or less than two options, these are the possible mentoring session types you have available